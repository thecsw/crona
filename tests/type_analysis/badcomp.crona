fn : int(a : int) {
    return true;
}

test : void() {
    a : int;
    a = (1== fn(true));
    return a;
}