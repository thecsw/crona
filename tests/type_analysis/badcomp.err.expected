FATAL [2,12]: Bad return value
FATAL [7,17]: Type of actual does not match type of formal
FATAL [7,7]: Invalid assignment operation
FATAL [8,12]: Return with a value in void function
Type Analysis Failed